zebra/zebra_l2.h

```diff
@@ -13,6 +13,7 @@
 #include "vlan.h"
 #include "vxlan.h"
 #include "zebra/zebra_vrf.h"
+#include <linux/if_bridge.h>
```

Comment:
can we use a linux-specific file here like this?

Reply:
I believe Mark is correct here as well.  Please move the <linux/if_bridge.h> into the appropriate .c files.  They should be limited to just a couple of places and it is useful to tell us if something is included right or not.

Reply:
If we are to use only vids directly, I think we can get rid of this